inst/extdata/ex-42-out/joined.tsv

```diff
@@ -0,0 +1,57 @@
+InputFile	SeqMD5Digest	SLength	Analysis	DB.ID	SignDesc	StartLoc	StopLoc	Score	Status	RunDate	IPRAcc	IPRDesc
```

Comment:
@jakekrol, do we have a column for Accessions (from NCBI/Uniprot) -- attaching with alphanumeric code will make it difficult to add metadata with our `acc2x` functions. thoughts?

Reply:
We could cut from the second "_" onward to set an accession number column. Again, I want to make sure we are careful in only using the accession number when necessary to prevent downstream effects.

Yeah, the alphanumeric code is to handle duplicate accession/header submissions.